bitnami/node/templates/pvc.yaml

```diff
@@ -1,22 +1,32 @@
-{{- if .Values.persistence.enabled }}
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: {{ template "common.names.fullname" . }}-binding
+  name: {{ include "common.names.fullname" . }}
```

Comment:
Why rename the PVC name? I think it is unnecessary and will mess with existing deployments.

Reply:
Good catch, my bad